Digester/Algorithm.ts

```diff
@@ -1,7 +1,10 @@
-export type Algorithm = "SHA-1" | "SHA-256" | "SHA-384" | "SHA-512"
+import { isly } from "isly"
+
+export type Algorithm = typeof Algorithm.values[number]
 
 export namespace Algorithm {
-	export function is(value: Algorithm | any): value is Algorithm {
-		return value == "SHA-1" || value == "SHA-256" || value == "SHA-384" || value == "SHA-512"
-	}
+	export const values = ["SHA-1", "SHA-256", "SHA-384", "SHA-512"]
```

Comment:
`as const` otherwise `Algorithm` will just be type `string`.

Reply:
Also the array can then technically be changed.

Reply:
Thanks